src/storage/heap_file.c

```diff
@@ -7820,10 +7819,102 @@ heap_mvcc_get_for_delete (THREAD_ENTRY * thread_p, OID * oid, OID * class_oid, R
 			  HEAP_SCANCACHE * scan_cache, int ispeeking, int old_chn, MVCC_REEV_DATA * mvcc_reev_data,
 			  NON_EXISTENT_HANDLING non_ex_handling_type)
 {
+  HEAP_GET_CONTEXT context;
+  SCAN_CODE scan;
+
```

Comment:
@eseokoh 

Yes, `heap_mvcc_get_for_delete` will be removed.

And `heap_mvcc_get_for_delete_new` should become a locator function. Currently there is an issue with multitable update and delete and I focused on this.